src/basic/architecture.h

```diff
@@ -20,6 +20,7 @@ typedef enum {
         ARCHITECTURE_ARM_BE,
         ARCHITECTURE_CRIS,
         ARCHITECTURE_IA64,
+        ARCHITECTURE_LOONGARCH64,
```

Comment:
needs an entry in the string table too. Also see #21288

Reply:
I just dropped the commit since #21288 will add it. This now depends on #21288 though as it won't compile without that landed.